icu4c/source/common/unicode/ubrk.h

```diff
@@ -132,6 +132,12 @@ typedef enum UBreakIteratorType {
  */
 #define UBRK_DONE ((int32_t) -1)
 
+/** Value indicating the text boundaries is open punctuation.
```

Comment:
any changes into {common,i18n}/unicode/*.h are public header and need API review to add them. and this is surely not true "stable ICU 2.0" Please move such into your .cpp file instead of put that into public header?

Reply:
Fixed